draft-trammell-taps-post-sockets.md

```diff
@@ -985,8 +985,6 @@ type Association interface {
 
 // A message together with with metadata needed to send it
 type OutMessage struct {
```

Comment:
IMO pretty much all of this gets nuked and replaced in my resolution to #3 (which I'll start as soon as all the pending pulls are merged and I've resynced on a coherent document)

Reply:
Sounds great.